moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -443,6 +453,64 @@ ompl::base::GoalPtr ompl_interface::ModelBasedPlanningContext::constructGoal()
   return ob::GoalPtr();
 }
 
+ompl::base::PlannerTerminationCondition ompl_interface::ModelBasedPlanningContext::constructPlannerTerminationCondition(
+    double timeout, const ompl::time::point& start)
+{
+  auto it = spec_.config_.find("termination_condition");
```

Comment:
Add code to the setup assistant to add this feature to the OMPL ``outputOMPLPlanningYAML()`` config file:
https://github.com/ros-planning/moveit/blob/master/moveit_setup_assistant/src/tools/moveit_config_data.cpp#L206

Reply:
It's an advanced, optional setting. Like `optimization_objective` it typically doesn't get set explicitly.

Reply:
Fair, as long as its in the tutorials